### DISCUSSION TITLE
Welcome to hotcakes-commerce-core Discussions!

### DISCUSSION THREAD

**Discussion by WillStrohl** [HotcakesCommerce/hotcakes-commerce-core#386] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Orrouk** [comment]
Hi, 
Many thanks to Will for creating this discussion, it's exactly what I was looking for. 
I installed and setup HCC 2 weeks ago. I'm a content creator used to work with dnn, not a developer nor a webstore expert; nevertheless, today, as far as I can see, everything is working fine. Again, thanks to Will for the tutorials. I'll come back soon with more feedback.

**Reply by WillStrohl** [reply]
Thank you for the kind note. :)
